ext/mysqli/tests/bind_fetch/datetime_types.phpt

```diff
@@ -0,0 +1,101 @@
+--TEST--
+mysqli bind_result datetimes
+--EXTENSIONS--
+mysqli
+--SKIPIF--
+<?php
+require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
+mysqli_check_skip_test();
+?>
+--FILE--
+<?php
+require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
+
+    $link = default_mysqli_connect();
+
+    // To get consistent result without depending on the DB version/setup
+    mysqli_query($link, "SET sql_mode=''");
+
+try {
+    mysqli_query(
+        $link,
+        "CREATE TABLE test_bind_result_datetime(
+            c1 date,
+            c2 time,
+            c3 timestamp(14),
+            c4 year,
+            c5 datetime,
+            c6 timestamp(4),
+            c7 timestamp(6)
+        ) ENGINE=" . get_default_db_engine()
+    );
+} catch (\mysqli_sql_exception) {
+    /* 14 Too big precision for timestamp */
```

Comment:
This is rather redundant, don't you think? Why are we executing a broken query that we know will fail? I might be missing the intention behind this, but even if the first query succeeded on some MySQL version, the expected output would not match. I think it's safe to delete it and keep only the correct one.

Reply:
Yeah I didn't get the point of this, and also noticed that it there was some overlap between tests. Will change this